specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Qualitative confidence measure.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "List element references.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        }
+      }
+    },
+    "FieldValueType": {
+      "type": "string",
+      "description": "Type of field value.",
+      "enum": [
+        "string",
+        "date",
+        "time",
+        "phoneNumber",
+        "number",
+        "integer",
+        "array",
+        "object"
+      ],
+      "x-ms-enum": {
+        "name": "FieldValueType",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },    
+    "Confidence": {
+      "description": "Confidence value.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1
+    },
+    "CopyRequest": {
+      "description": "Contract to initiate a copy request.",
+      "type": "object",
+      "properties": {
+        "targetInformation": {
+          "description": "Information about target subscription.",
+          "type": "object",
+          "properties":{
+            "endpoint": {
+              "description": "Get or set endpoint path.",
+              "maxLength": 2048,
+              "minLength": 0,
+              "type": "string"
+            },
+            "resourceId": {
+              "description": "Get or set resource identifier.",
+              "maxLength": 2048,
+              "minLength": 0,
+              "type": "string"
+            }
+          }
+        },
+        "copyConfig": {
+          "description": "Copy operation options.",
+          "type": "object",
+          "required":[
+            "deleteSource"
+          ],
+          "properties":{
+            "deleteSource": {
+              "description": "Indicate deletion of source model artifacts.",
+              "type":"boolean",
+              "default":false
+            },
+            "supportCrossTenantCopy": {
+              "description": "Indicate copy target tenancy requirement.",
+              "type": "boolean",
+              "default":false
+            },
+            "targetModelMetadata": {
+              "description": "Metadata for copied model to Target",
+              "type": "object",
+              "properties":{
+                "name": {
+                  "description": "Name of copied model at Target.",
+                  "maxLength": 128,
+                  "minLength": 0,
+                  "type": "string"
+                },
+                "description": {
+                  "description": "Description of copied model.",
+                  "maxLength": 1024,
+                  "minLength": 0,
+                  "type": "string"
+                },
+                "tags": {
+                  "description": "Tags for copied model",
```

Comment:
What is "tags"?

Reply:
Tracked here - https://github.com/Azure/azure-rest-api-specs/pull/7303#discussion_r331674875